atc/plan.go

```diff
@@ -111,16 +111,25 @@ type PutPlan struct {
 }
 
 type CheckPlan struct {
-	Type        string  `json:"type"`
-	Name        string  `json:"name,omitempty"`
-	Source      Source  `json:"source"`
-	Tags        Tags    `json:"tags,omitempty"`
-	Timeout     string  `json:"timeout,omitempty"`
-	FromVersion Version `json:"from_version,omitempty"`
+	Type        string            `json:"type"`
+	Name        string            `json:"name,omitempty"`
+	Source      Source            `json:"source"`
+	Tags        Tags              `json:"tags,omitempty"`
+	Timeout     string            `json:"timeout,omitempty"`
+	FromVersion Version           `json:"from_version,omitempty"`
+	SpanContext map[string]string `json:"span_context"`
```

Comment:
Do we need this here if we're storing the span_context in its own column?

Reply:
Ah yes you're right we can take that out now!